### PR TITLE
Updates content

### PR DESCRIPTION
**Pull request by emileighoutlaw**
Touts eFiling data and statutes. 
Updates date for release notes.

cc @porta-antiporta and @noahmanger for content. Let me know if you see edits that are needed.

### CODE REVIEW

**Comment by porta-antiporta** [comment]
@emileighoutlaw because `advisory opinions` is also relatively new, can we include that as well?

**Comment by emileighoutlaw** [comment]
For sure! Coming right up

File: fec/home/templates/home/home_page.html [hunk 1]
```diff
@@ -52,18 +52,18 @@ <h2 class="feature__title">New features</h2>
               <img class="card__image" src="{% static "img/i-complex--data.svg" %}" alt="Icon representing explore campaign finance data">
             </a>
             <div class="card__content">
-              <p>Download itemized <a href="{{ settings.FEC_APP_URL }}/receipts/?two_year_transaction_period=2016&is_individual=true">receipts</a> and <a href="{{ settings.FEC_APP_URL }}/disbursements/?two_year_transaction_period=2016">disbursements</a>.</p>
+              <p>Searchable <a href="/data/filings?data_type=efiling">eFiling data</a>, which contains the most recent, raw information from committee electronic filings.</p>
```
Comment (ccostino) [inline fec/home/templates/home/home_page.html]: Does this URL need the `{{ settings.FEC_APP_URL }}` that was here before?

**Comment by emileighoutlaw** [comment]
That link should be right, @ccostino :)

**Comment by ccostino** [comment]
Thank you, @emileighoutlaw!

**Pull request merged**